accountgw_test.go

```diff
@@ -26,58 +27,75 @@ func TestGatewayAccount_EstimateAndExecute(t *testing.T) {
 	testSet := map[string][]testSetType{
 		"devnet": {{
 			ExecuteCalls: []types.FunctionCall{{
```

Comment:
Ideally we should use the rpcv02 types. Also, why were the ContracAddresses removed?

Reply:
Didn't remove the ContractAddress, just replaced the repetition of the same code to a loop.
Will check out the rpcv02 types.